src/main/java/com/fusionflux/portalcubed/client/render/block/entity/RocketTurretRenderer.java

```diff
@@ -48,10 +51,8 @@ public void render(RocketTurretBlockEntity entity, float tickDelta, MatrixStack
         matrices.scale(-1, -1, 1);
         matrices.translate(0.5, -1.501, -0.5);
 
-        final BlockEntityWrapperEntity<RocketTurretBlockEntity> wrapper = new BlockEntityWrapperEntity<>(entity);
```

Comment:
Why did you change this to use another `WeakHashMap`?

Reply:
Because this was creating a new `BlockEntityWrapperEntity` every single frame

Reply:
That's still significantly cheaper than doing a lookup (and simpler too). Additionally, if you're arguing about allocations, that lambda will be an allocation every frame because you're capturing a local (`entity`).

Reply:
I think it would be better to keep the wrapper lookup and store the model on the wrapper.

Reply:
You would have to explain what you mean by store the model on the wrapper

Reply:
Instead of having two separate `WeakHashMap`s for the wrappers and for the models, the models could go in the wrappers.

Reply:
Ah.